### ISSUE TITLE
Cannot find module 'next-international/middleware'

### ISSUE THREAD

**Issue by nodgear** [QuiiBz/next-international#58]
Super happy to see it got /app support.

Uppon checking the example turns out i cannot import the middleware and other objects.

```bash
// [lang]/layout
Module '"next-international"' has no exported member 'I18nProviderServer'.

//middleware
Cannot find module 'next-international/middleware' or its corresponding type declarations.
```

I can surpress the issue by importing from `/dist/middleware` but then i get another error about it not being a named export there.

Package Version: 0.5 from the latest release tag.

**Comment by turgayaksu** [comment]
use as below
`import { createI18nMiddleware } from 'next-international/dist/app/middleware';`

**Comment by QuiiBz** [comment]
Thanks, I've just released 0.5.1 which fix this issue: https://github.com/QuiiBz/next-international/releases/tag/0.5.1
